packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -1141,7 +1137,7 @@ export class IndexedDbLruDelegate implements ReferenceDelegate, LruDelegate {
       .removeTargets(txn, upperBound, activeTargetIds);
   }
 
-  removeMutationReference(
+  markPotentiallyOrphaned(
```

Comment:
Renamed this method as I need to use it to trigger `key` to be considered potentially orphaned. Only then with Eager GC pick it up during its GC run. Other names welcome, including `removeMutationReference`, which reveals less implementation details.